aptos-move/move-examples/token_objects/ambassador/client/ambassador.ts

```diff
@@ -0,0 +1,171 @@
+// Copyright © Aptos Foundation
+// SPDX-License-Identifier: Apache-2.0
+
+import { AptosAccount, AptosClient, HexString } from "aptos";
+
+const NODE_URL = process.env.APTOS_NODE_URL || "https://fullnode.testnet.aptoslabs.com";
```

Comment:
nit: Use these values from common.ts instead

Reply:
This Typescript client example is in `aptos-move/move-examples/token_objects/ambassador/` (near the Move contracts), not in the same directory where `common.ts` is.

Reply:
Hmm I didn't notice this. I think we should move ambassador.ts there under ecosystem/ instead of here. This is how we structure tutorials - the move modules in aptos-move/move-examples while the TS parts are in ecosystem/

Reply:
I see. I was following the example of `https://github.com/aptos-labs/aptos-core/tree/main/aptos-move/move-examples/my_first_dapp`. OK, I will move the TS parts into `ecosystem`.